syncopy/datatype/base_data.py

```diff
@@ -337,8 +337,17 @@ def _set_dataset_property_with_dataset(self, inData, propertyName, ndim):
         setattr(self, "_" + propertyName, inData)
 
     def _set_dataset_property_with_list(self, inData, propertyName, ndim):
-        """
-        Coming soon...
+        """Set a dataset property with list of NumPy arrays
+
+        Parameters
+        ----------
+            inData : list
```

Comment:
Hm, you're right: `list` here really just serves as a container for NumPy arrays. So yes, it doesn't make a difference if the arrays are packed into a `tuple`, `set` or any other iterable. I just picked list since we can easily check for it in `_set_dataset_property` Do you

Reply:
I think one could check for iterables with sth like `bool(iter(my_container))`, but I guess it's not critical and lists are the go-to containers for such cases imho!